### PR TITLE
fix(v2): allow search page swizzle

### PR DESCRIPTION
**Pull request by Ako92**
## Motivation

To Improve SearchPage to make swizzle cli work on it.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

I Did it.

## Test Plan

Naviagate to website directory and run:
``` npm run swizzle @docusaurus/theme-search-algolia SearchPage ```

### CODE REVIEW

File: packages/docusaurus-theme-classic/src/index.js [hunk 1]
```diff
@@ -128,6 +128,7 @@ const swizzleAllowedComponents = [
   'SearchBar',
   'hooks/useTheme',
   'prism-include-languages',
+  'SearchPage',
```
Comment (anshulrgoyal) [inline packages/docusaurus-theme-classic/src/index.js:131]: add it to search plugin itself.
```js
module.exports.getSwizzleComponentList = () =>['SearchPage']
```

Comment (Ako92) [reply packages/docusaurus-theme-classic/src/index.js:131]: what do you mean?
where should i add this?
this file?
image

Comment (anshulrgoyal) [reply packages/docusaurus-theme-classic/src/index.js:131]: in` docusaurus-theme-search-algolia/index.js`

Comment (slorber) [reply packages/docusaurus-theme-classic/src/index.js:131]: @anshulrgoyal for now I think we should just allow swizzle, not recommend it, as it's likely we change a bit the implementation of this page, so not adding this array is fine

Comment (slorber) [reply packages/docusaurus-theme-classic/src/index.js:131]: But yes it shuld not be added to this list in the classic theme anyway

Comment (slorber) [inline packages/docusaurus-theme-classic/src/index.js:131]:
```suggestion
```

**Pull request merged**